packages/mdl-base/README.md

```diff
@@ -0,0 +1,241 @@
+# mdl-base
+
+> NOTE: Please do not use or rely on `adapter.js`. It is deprecated and in the process of being
+> removed.
+
+MDL base contains core foundation and component classes that serve as the base classes for all of MDL's foundation classes and components (respectively).
+
+Most of the time, you shouldn't need to depend on `mdl-base` directly. It is useful however if you'd like to write custom components that follow MDL's pattern and elegantly integrate with the MDL ecosystem.
+
+## Installation
+
+First install the module:
+
+> NOTE: Installation via the npm registry will be available after alpha.
+
+Then include it in your code in one of the following ways:
+
+#### ES2015+
+
+```javascript
+import MDLBase, {MDLBaseFoundation} from 'mdl-base';
```

Comment:
I think I would lean towards `MDL{Component,Foundation}` unless we are planning to use them for something else. Although I understand there's a bit of a subtle discrepancy with the package/class naming (classes in base wouldn't necessarily be prefixed by base, whereas generally for components they would be prefixed by the component name). Though, I like the idea of keeping the focus on the main keywords {Component, Foundation} throughout the docs.  (base isn't really a concept in itself)